src/main/java/com/twilio/twiml/Gather.java

```diff
@@ -47,6 +65,10 @@
     @XmlAttribute
     private final String acknowledgeSoundUrl;
 
+    @XmlAttribute
+    @XmlJavaTypeAdapter(TwiML.ToStringAdapter.class)
+    private final Input input;
```

Comment:
Probably just want this to be a string so we can support space-separated input types. e.g. `dtmf speech`.